analyzer/src/main/java/org/jboss/gm/analyzer/alignment/io/SettingsFileIO.java

```diff
@@ -0,0 +1,93 @@
+package org.jboss.gm.analyzer.alignment.io;
+
+import static org.apache.commons.lang.StringUtils.isEmpty;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.io.FileUtils;
+import org.commonjava.maven.ext.common.ManipulationUncheckedException;
+import org.gradle.api.logging.Logger;
+import org.jboss.gm.common.logging.GMLogger;
+
+public class SettingsFileIO {
```

Comment:
Moved all the settings.gradle code into a separate utility class to prevent everything going into alignment task

Reply:
:+1:

Reply:
Super minor nitpick, the class should probably be `final`

Reply:
Done :-)